app/src/navigation/screens/VendorDetails.tsx

```diff
@@ -130,12 +135,24 @@ const VendorDetailsScreen: React.FC = () => {
           onPress={async () => {
             if (vendor) {
```

Comment:
Recommend using guard clauses to prevent overabundance of indentation -- good for readability